music_service_async_interface/__init__.py

```diff
@@ -41,14 +86,12 @@ async def object_from_url(self, url: str) -> "Object":
     @staticmethod
     @abstractmethod
     def is_valid_url(url) -> bool:
-        """
-        Performs basic check if string looks like music service URL
-        """
+        """Performs basic check if string looks like music service URL"""
         ...
 
     async def parse_urls(self, long_string: str) -> AsyncGenerator["Object", None]:
-        """
-        Parses `long_string` including music service URLs to corresponding music service objects
+        """Parses `long_string` including music service URLs to corresponding music service objects
```

Comment:
why?

Reply:
This style is used officially and I was mixing both of them, so I replaced them to one.
I want it to be first line of docstring, not second.

Reply:
For example in this block: https://pythonhosted.org/an_example_pypi_project/sphinx.html#format_exception